ci-operator/step-registry/crio-test/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- wgahnagl
```

Comment:
we don't need this anymore do we?

Reply:
yeah all of the crio-test stuff can go away methinks

Reply:
the crio-test stuff is where all the tests are, actually lol 
should we rename it to just crio?